NEWS.rst

```diff
@@ -1,6 +1,13 @@
 History
 =======
 
+0.9 (2019-10-02)
```

Comment:
My latest PR (empty target strings) is missing. Git log also mentions something about conversion from PSRCAT.

Reply:
I only tend to put majorish PRs into NEWS.rst... #53 is a minor fix while #16 will probably have no interested users 😄